plugin/src/main/java/com/spotify/plugin/dockerfile/AbstractDockerMojo.java

```diff
@@ -192,6 +192,12 @@ public String getFileName() {
   private JarArchiver jarArchiver;
 
   /**
+   * Allows enabling/disabling gcr credentials
+   */
+  @Parameter(defaultValue = "true", property = "dockerfile.googleregistry")
```

Comment:
If dockerfile.googleregistry property doesn't exist, shouldn't this value have false as default? If that's the case you don't even need to specify defaultValue in the Parameter annotations.